data/presets/leisure/parklet.json

```
@@ -0,0 +1,28 @@
+{
+    "icon": "maki-picnic-site",
+    "fields": [
+        "name",
+        "parklet/seatings",
+        "parklet/greenery",
+        "material",
+        "colour",
+        "seasonal"
+    ],
+    "moreFields": [
+        "level",
+        "manufacturer",
+        "operator",
+        "wheelchair"
+    ],
```

```suggestion
    "fields": [
        "parklet/seatings",
        "parklet/greenery",
        "material",
        "colour",
        "seasonal"
    ],
    "moreFields": [
        "name",
        "level",
        "manufacturer",
        "operator",
        "wheelchair"
    ],
```

Are there parklets out there with a valid name tag? I fear this will be miss used because it is the first thing this preset shows. Maybe move it into `moreFields` to just show what is there?